### PR TITLE
link color changes

### PR DESCRIPTION
**Pull request by drewnielson**

### CODE REVIEW

File: docs/testfiles/TC07-catchall/07-visitedlinkcolors.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Visited/Unvisited Link Color Samples</title>
+    <link rel="stylesheet" href="linkcolors.css">
+  </head>
+  <body>
+    <div class="container">
+      <h1>Visited/Unvisited Link Samples</h1>
```
Comment (kengdoj) [inline docs/testfiles/TC07-catchall/07-visitedlinkcolors.html:10]: change to h2

File: docs/testfiles/TC07-catchall/07-visitedlinkcolors.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        </p>
+        <h2>Code Sample Index - Use of color in links</h2>
+          <h3>Key</h3>
+            <table>
+              <tr>
+                <th scope="row">
+                  PASS
+                </th>
+                <td>
+                  Use of color plus some other method
```
Comment (kengdoj) [inline docs/testfiles/TC07-catchall/07-visitedlinkcolors.html:22]: color plus font style

Comment (drewnielson) [reply docs/testfiles/TC07-catchall/07-visitedlinkcolors.html:22]: fixed as suggested

File: docs/testfiles/TC07-catchall/07-visitedlinkcolors.html (hunk 1 above)
Comment (drewnielson) [reply docs/testfiles/TC07-catchall/07-visitedlinkcolors.html]: Fixed as suggested

File: docs/testfiles/TC07-catchall/07-visitedlinkcolors.html (hunk 2 above)
Comment (drewnielson) [reply docs/testfiles/TC07-catchall/07-visitedlinkcolors.html:22]: Fixed as suggested

**Comment by drewnielson** [comment]
@kengdoj take another look

**Pull request closed without merging**